utility/util.js

```diff
@@ -0,0 +1,34 @@
+function findOverlap(country1, country2){
+    var countryOne = country1['noVisa']
+    var countryTwo = country2['noVisa']
+    var overlap = []
+    for (i=0; i<countryOne.length; i++){
+        countryTwo.includes(countryOne[i]) ? overlap.push(countryOne[i]) : null
+    }
+}
+function formatCountryText(response){
+    var officialCountry = {
+        'noVisa': [],
+        'visa': [],
+        'depends': []
+    }
+    var countries = response.split('<tr>')
+    countries.forEach((country) => {
+        if (country.split('<td')[1] && !country.includes('Visa_policy_by_country')) {
+
+            var firstLine = country.split('<td')[1];
+            if (firstLine.match(/title="(.*)"/g)) {
+                var countryName = firstLine.match(/title="(.*)"/g)[0].split("\"")[1];
+                if (country.includes('table-yes')) {
+                    officialCountry['noVisa'].push(countryName)
```

Comment:
when we know what column we are adding something to we should use . notation.

Reply:
When would be pushing and not know which column we are adding to though?

Reply:
I'm just saying this should say
```
officialCountry.noVisa.push(countryName)
```